tests/tests_lite/conftest.py

```diff
@@ -83,6 +85,46 @@ def reset_deterministic_algorithm():
     torch.use_deterministic_algorithms(False)
 
 
+def mock_cuda_count(monkeypatch, n: int) -> None:
+    monkeypatch.setattr(lightning_lite.accelerators.cuda, "num_cuda_devices", lambda: n)
+    monkeypatch.setattr(lightning_lite.accelerators.cuda, "num_cuda_devices", lambda: n)
+
+
+@pytest.fixture(scope="function")
+def cuda_count_0(monkeypatch):
+    mock_cuda_count(monkeypatch, 0)
+
+
+@pytest.fixture(scope="function")
+def cuda_count_1(monkeypatch):
+    mock_cuda_count(monkeypatch, 1)
+
+
+@pytest.fixture(scope="function")
+def cuda_count_2(monkeypatch):
+    mock_cuda_count(monkeypatch, 2)
+
+
+@pytest.fixture(scope="function")
+def cuda_count_4(monkeypatch):
+    mock_cuda_count(monkeypatch, 4)
+
+
+def mock_mps_count(monkeypatch, n: int) -> None:
```

Comment:
@awaelchli You added these to Lite conftest but they are unused. I suggest removing them.

Reply:
Introducing these inconsistencies in the two code bases generates a lot of new work when moving/rewriting tests to Lite.

Reply:
We can add them to Lite too if it reduces the boilerplate. Just saying that either we add them and use them or not add them. When I made the comment above, they were added but not used.

I see now you have removed them. We can add them and refactor the lite tests in a different PR if you would like.